acslib/ccure/crud.py

```diff
@@ -83,22 +71,8 @@ def update(self, personnel_id: int, update_data: dict) -> ACSRequestResponse:
         :param personnel_id: the Personnel object's CCure ID
         :param update_data: maps Personnel properties to their new values
         """
-        return self.connection.request(
-            ACSRequestMethod.PUT,
-            request_data=ACSRequestData(
-                url=self.config.base_url + self.config.endpoints.EDIT_OBJECT,
-                params={
-                    "type": "SoftwareHouse.NextGen.Common.SecurityObjects.Personnel",
-                    "id": personnel_id,
-                },
-                data=self.connection.encode_data(
-                    {
-                        "PropertyNames": list(update_data.keys()),
-                        "PropertyValues": list(update_data.values()),
-                    }
-                ),
-                headers=self.connection.base_headers | self.connection.header_for_form_data,
-            ),
+        return super().update(
```

Comment:
I'm putting the comment here, since there is no change above. The method arguments don't agree with the inherited definition.

Reply:
I've changed `personnel_id` to `object_id` to match the inherited method's arg name, but I haven't added an `object_type` arg to `personnel.update`. Should it be required that the user specify the object type like `ccure.personnel.update(object_type="personnel", object_id=5001, update_data={})`? Should object type be just assumed there?